src/app/games/connect-six/tests/ConnectSixMinimax.spec.ts

```diff
@@ -0,0 +1,169 @@
+import { Coord } from 'src/app/jscaip/Coord';
+import { ConnectSixMinimax } from '../ConnectSixMinimax';
+import { ConnectSixDrops, ConnectSixFirstMove, ConnectSixMove } from '../ConnectSixMove';
+import { ConnectSixNode, ConnectSixRules } from '../ConnectSixRules';
+import { ConnectSixState } from '../ConnectSixState';
+import { PlayerOrNone } from 'src/app/jscaip/Player';
+import { Table } from 'src/app/utils/ArrayUtils';
+
+/* eslint-disable max-lines-per-function */
+describe('ConnectSixMinimax', () => {
+
+    let minimax: ConnectSixMinimax;
+
+    const _: PlayerOrNone = PlayerOrNone.NONE;
+    const O: PlayerOrNone = PlayerOrNone.ZERO;
+
+    beforeEach(() => {
+        const rules: ConnectSixRules = ConnectSixRules.get();
+        minimax = new ConnectSixMinimax(rules, 'Minimax');
+    });
+    it('should propose only one move at first turns', () => {
+        // Given the minimax and the ConnectSix size of board, and the initial board
```

Comment:
le "size of board" c'est pas vraiment important comme given, tu pourrais dire "given a board" simplement, puis plus tard, "it should be in the center of the board" est clair